content/en/docs/instrumentation/python/automatic.md

```diff
@@ -88,6 +92,15 @@ The examples that follow send instrumentation results to the console. Learn more
 about installing and configuring the [OpenTelemetry Distro](../distro) to send
 telemetry to other destinations, like an OpenTelemetry Collector.
 
+**Note**: If you were to use auto-instrumentation and manually instrumenting your code the
+pacakage `opentelemetry-distro` won't let you set a `Resource` with attributes defined in your code, 
```

Comment:
I would clarify this little. It's more like distro already sets up the tracing pipeline for you and you can't override the global tracer provider. But end users have the capability to create more than one provider with different resource attributes.